src/model/core/transmission/transmission.jl

```diff
@@ -219,32 +160,14 @@ function transmission!(EP::Model, inputs::Dict, setup::Dict)
 
 	### Constraints ###
 
-	if MultiStage == 1
-		# Linking constraint for existing transmission capacity
-		@constraint(EP, cExistingTransCap[l=1:L], vTRANSMAX[l] == inputs["pTrans_Max"][l])
-	end
-
   	## Power flow and transmission (between zone) loss related constraints
 
 	# Maximum power flows, power flow on each transmission line cannot exceed maximum capacity of the line at any hour "t"
-	# Allow expansion of transmission capacity for lines eligible for reinforcement
 	@constraints(EP, begin
-		cMaxFlow_out[l=1:L, t=1:T], vFLOW[l,t] <= eAvail_Trans_Cap[l]
-		cMaxFlow_in[l=1:L, t=1:T], vFLOW[l,t] >= -eAvail_Trans_Cap[l]
+		cMaxFlow_out[l=1:L, t=1:T], vFLOW[l,t] <= EP[:eAvail_Trans_Cap][l]
+		cMaxFlow_in[l=1:L, t=1:T], vFLOW[l,t] >= -EP[:eAvail_Trans_Cap][l]
 	end)
 
-	# If network expansion is used:
```

Comment:
What's with the large number of things deleted in this file?

Reply:
We basically moved around things for ease of code maintenance and keeping in view that in GenX v0.4.1 (or, maybe v0.4.2) we will need to have the DC-OPF based transmission expansion included (which requires merging of Filippo's Bender's decomposition; So, future agenda)

Reply:
So, everything from this file pertaining to transmission expansion has been moved to investment_transmission.jl. Currently, that's still based off of the non-DC-OPF flows. In the next version, in addition to this, there will also be a DC-OPF based investment_transmission.

Reply:
Those lines are in investment_transmission.jl now, so we separate investment and operation, just like we do for resources and storage.